ui-components/src/commonMain/kotlin/org/jetbrains/kotlinconf/ui/components/MarkdownScreenWithTitle.kt

```diff
@@ -0,0 +1,105 @@
+package org.jetbrains.kotlinconf.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.mikepenz.markdown.compose.Markdown
+import com.mikepenz.markdown.model.DefaultMarkdownColors
+import com.mikepenz.markdown.model.DefaultMarkdownTypography
+import kotlinconfapp.ui_components.generated.resources.Res
+import kotlinconfapp.ui_components.generated.resources.arrow_left_24
+import kotlinconfapp.ui_components.generated.resources.main_header_back
+import org.jetbrains.compose.resources.stringResource
+import org.jetbrains.kotlinconf.ui.theme.KotlinConfTheme
+
+@Composable
+fun MarkdownScreenWithTitle(
```

Comment:
Since this is not an individual component but really an entire screen, I'd say it's not part of the design system but the app code. It's great to have it extracted and reused like this, I'd just move it to the `shared` module instead of having it in `ui-components`.